application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageAutoDetection.java

```diff
@@ -63,6 +68,13 @@ public void onMessageReceived(MessageReceivedEvent event) {
         codeMessageHandler.addAndHandleCodeMessage(originalMessage, true);
     }
 
+    private boolean isSentByIgnoredMember(Member member) {
```

Comment:
Yeah, didn't want to nitpick on the whole 'ignored' theme. But the member is not being ignored, it's more that the role is exempt or immune to this feature.

But I'm fine with this, you can _ignore_ it. ^^

Just unsure if the reader would know why we are ignoring some members, naming doesn't communicate well our intentions.

Reply:
isSentByIgnoredMember.. -> isSentByExcludedRole..

Reply:
Yeah, I think I like that more, thanks.

You probably want to update everything _ignored_ to _excluded_. For example, `isExcludedRolePattern` etc, so it's consistent.

Reply:
done!